pkg/drivers/http/client.go

```diff
@@ -35,11 +35,12 @@ func NewClient(isSecure bool, basicAuth *BasicAuth, cert string) (Client, error)
 	httpClient := &http.Client{Transport: http.DefaultTransport}
 	if isSecure {
 		if cert == "" {
-			/* #nosec */
+			//nolint:gosec
 			httpClient.Transport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
 		} else {
 			caCertPool := x509.NewCertPool()
 			caCertPool.AppendCertsFromPEM([]byte(cert))
+			////nolint:gosec
```

Comment:
Does the linter directive work with 4 slashes?

Reply:
well it works :) but i will stick to 2.